assets/css/src/components/_featureAnnouncement.scss

```diff
@@ -23,3 +23,34 @@
   top: -4px;
   width: 10px;
 }
+
+.mailpoet_update_notice {
```

Comment:
Just a suggestion: What about name `.mailpoet_in_beamer_update_notice` I think that this name could help explain why such a big z-index is needed.